dev/scalastyle-config.xml

```diff
@@ -242,7 +242,7 @@ This file is divided into 3 sections:
       <parameter name="groups">java,scala,org,apache,3rdParty,comet</parameter>
       <parameter name="group.java">javax?\..*</parameter>
       <parameter name="group.scala">scala\..*</parameter>
-      <parameter name="group.org">org\.(?!apache\.comet).*</parameter>
```

Comment:
We should exclude all apache packages from the `org` group, not just comet. We have a separate rule below for excluding comet from the apache group.

Without this change it was not possible to satisfy both scalastyle and scalafix at the same time as they wanted different import orders.